VirtoCommerce.PricingModule.Web/Scripts/blades/assignment-list.tpl.html

```diff
@@ -12,6 +23,8 @@
                         { name: 'pricelist.name', displayName: 'pricing.blades.assignment-list.labels.pricelist' },
                         { name: 'pricelist.currency', displayName: 'pricing.blades.assignment-list.labels.pricelist-currency' },
                         { name: 'priority', displayName: 'pricing.blades.assignment-list.labels.priority', width: 72, sort: { direction: uiGridConstants.DESC } },
+                        { name: 'startDate', displayName: 'pricing.blades.assignment-list.labels.activation', width: 87 },
+                        { name: 'endDate', displayName: 'pricing.blades.assignment-list.labels.expiration', width: 87 },
```

Comment:
Please do not forget to fix these columns appearance and attach a demo.

Reply:
It could be some caching issue, in my case they've appeared after some time only

Reply:
Seems that if first row does not have these values (startDate & endDate), they are not shown in the columns, and this change has no effect in that scenario.
Reason: https://github.com/VirtoCommerce/vc-platform/blob/853d66a976c47bf8d7b2c5222c6acf2bfc720915/VirtoCommerce.Platform.Web/Scripts/common/uiGridUtils.js#L97
Need to solve that situation somehow. Also when first row has values for startDate or endDate, corresponding column becomes visible by default. It should not

Reply:
Cannot reproduce described behaviour:
![image](https://user-images.githubusercontent.com/42266842/53359992-7467d800-3945-11e9-96f3-59aa19b1924c.png)
![ezgif com-optimize 1](https://user-images.githubusercontent.com/42266842/53361270-6ebfc180-3948-11e9-8d0c-9bd11085bc20.gif)

Could you please tell reproduce steps more detailed (or show demo)?

Reply:
Indeed, described behavior reproduces if you have grid options without these dates in localStorage in
"ngStorage-gridState:Modules/$(VirtoCommerce.Pricing)/Scripts/blades/assignment-list.tpl.html".
If clear it - it works as expected.
Probably we do not need to handle such specific situation here.